src/nectarchain/dqm/pixel_participation.py

```diff
@@ -0,0 +1,148 @@
+from dqm_summary_processor import DQMSummary
+from matplotlib import pyplot as plt
+from ctapipe.visualization import CameraDisplay
+from ctapipe.instrument import CameraGeometry
+from ctapipe.coordinates import EngineeringCameraFrame
+import numpy as np
+
+
+class PixelParticipation_HighLowGain(DQMSummary):
+    def __init__(self, gaink):
+        self.k = gaink
+        return None
+
+    def ConfigureForRun(self, path, Pix, Samp, Reader1):
+        # define number of pixels and samples
+        self.Pix = Pix
+        self.Samp = Samp
+
+        self.counter_evt = 0
+        self.counter_ped = 0
+
+        self.camera = CameraGeometry.from_name("NectarCam-003").transform_to(EngineeringCameraFrame())
+        self.camera2 = CameraGeometry.from_name("NectarCam-003").transform_to(EngineeringCameraFrame())
+
+        self.cmap = "gnuplot2"
+        self.cmap2 = "gnuplot2"
+
+
+        self.BadPixels_ped = np.zeros(self.Pix)
```

Comment:
Please declare all class members within `__init__`.

Reply:
Is it ok if I fix the issues related to pixel_participation and pixel_timeline in this PR and then I do another PR to work on the format of the code.

Reply:
There is no problem in defining the members in `ConfigureForRun`, I was refereeing to their declaration.

Reply:
Done